src/librbd/ExclusiveLock.h

```diff
@@ -40,15 +30,12 @@ class ExclusiveLock {
   void try_lock(Context *on_tried_lock);
   void request_lock(Context *on_locked);
   void release_lock(Context *on_released);
```

Comment:
Minor: no need for these pass-through methods -- just let the ```ManagedLock``` version be exposed